build/python/build/cmake.py

```
@@ -133,11 +133,13 @@ class CmakeProject(Project):
     def __init__(self, url: Union[str, Sequence[str]], md5: str, installed: str,
                  configure_args: list[str]=[],
                  windows_configure_args: list[str]=[],
+                 android_configure_args: list[str]=[],
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)